INSTALL.md

```diff
@@ -163,18 +226,18 @@ add_app () {
 }
 
 # Set compiler
-export CC=gcc
-export CXX=g++
+export CC=${CC:-gcc}
+export CXX=${CXX:-g++}
```

Comment:
I would not put this into the example one. As a matter of fact I would remove it from the `KRATOS_SOURCE` as well.

This syntax is meant only for compiling in env where is more convenient to define envs outside the script, but makes no sense for people if they copy paste it